.gitignore

```diff
@@ -1,2 +1,7 @@
+
+#Libraries and modules
 node_modules
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+# Test output files
+iana.org
```

Comment:
Please make a separate branch and PR for the gitignore changes.

Reply:
Noted :)